tests/backup/backup_helper.go

```diff
@@ -979,22 +979,43 @@ func CreateRestoreWithValidation(ctx context.Context, restoreName, backupName st
 	return
 }
 
-func getSizeOfMountPoint(podName string, namespace string, kubeConfigFile string, volumeMount string) (int, error) {
+func getSizeOfMountPoint(podName string, namespace string, kubeConfigFile string, volumeMount string, containerNames ...string) (int, error) {
 	var number int
-	ret, err := kubectlExec([]string{fmt.Sprintf("--kubeconfig=%v", kubeConfigFile), "exec", "-it", podName, "-n", namespace, "--", "/bin/df"})
+	var str string
+	output, err := kubectlExec([]string{fmt.Sprintf("--kubeconfig=%v", kubeConfigFile), "exec", "-it", podName, "-n", namespace, "--", "/bin/df"})
 	if err != nil {
 		return 0, err
 	}
-	for _, line := range strings.SplitAfter(ret, "\n") {
+	for _, line := range strings.SplitAfter(output, "\n") {
 		if strings.Contains(line, volumeMount) {
-			ret = strings.Fields(line)[3]
+			str = strings.Fields(line)[3]
+			break
 		}
 	}
-	number, err = strconv.Atoi(ret)
-	if err != nil {
-		return 0, err
+	if str == "" {
+		log.Infof("Could not find any mount points for the volume mount [%s] in the pod [%s] in namespace [%s] ", volumeMount, podName, namespace)
+		log.Infof("Trying to check if there is a sym link for [%s]", volumeMount)
+		symlinkPath, err := core.Instance().RunCommandInPod([]string{"readlink", "-f", volumeMount}, podName, containerNames[0], namespace)
```

Comment:
Since containerNames is an optional argument how about we check the len of containerNames before we access values in it?

Reply:
Fixed it